database/migrations/2025_07_09_140813_apply-existing-admins-to-all-teams.php

```diff
@@ -1,15 +1,12 @@
 <?php
 
 use Illuminate\Database\Migrations\Migration;
-use Illuminate\Database\Schema\Blueprint;
```

Comment:
Just linting in this file.

Reply:
Going to need this to be removed from this as we'll be cherry-picking this and this command doesn't exist in main yet :)

Reply:
https://github.com/HDRUK/gateway-api/pull/1316/files